content/sensu-go/6.8/operations/maintain-sensu/disaster-recovery.md

```diff
@@ -0,0 +1,436 @@
+
+sensu-backend start \
+--etcd-initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--etcd-initial-cluster-token sensu-backend-03 \
+--etcd-initial-advertise-peer-urls http://sensu-backend-03:2380 \
+--etcd-advertise-client-urls http://sensu-backend-03:2379
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+4. Confirm that the new Sensu backend or cluster is running:
+
+   {{< code shell >}}
+systemctl status sensu-backend
+{{< /code >}}
+
+   The response should indicate `active (running)`.
+
+5. Copy the etcd snapshot file to each cluster member so that all members will be restored using the same snapshot.
+
+6. Restore the Sensu configuration from the etcd snapshot to the running backend or cluster:
+
+   {{% notice warning %}}
+**IMPORTANT**: Update the example values in these commands according to your Sensu configuration before running the commands.
+{{% /notice %}}
+
+   {{< language-toggle >}}
+
+{{< code shell "Restore a single backend" >}}
+ETCDCTL_API=3 etcdctl snapshot restore sensu_etcd_snapshot.db \
+--name sensu-backend-01 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--initial-cluster-token sensu-backend-01 \
+--initial-advertise-peer-urls http://sensu-backend-01:2380 \
+--data-dir /var/lib/sensu/sensu-backend/etcd/data \
+--wal-dir /var/lib/sensu/sensu-backend/etcd/wal 
+{{< /code >}}
+
+{{< code shell "Restore a clustered backend" >}}
+ETCDCTL_API=3 etcdctl snapshot restore sensu_etcd_snapshot.db \
+--name sensu-backend-01 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--initial-cluster-token sensu-backend-01 \
+--initial-advertise-peer-urls http://sensu-backend-01:2380 \
+--data-dir /var/lib/sensu/sensu-backend/etcd/data \
+--wal-dir /var/lib/sensu/sensu-backend/etcd/wal
+
+ETCDCTL_API=3 etcdctl snapshot restore snapshot.db \
+--name sensu-backend-02 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--initial-cluster-token sensu-backend-02 \
+--initial-advertise-peer-urls http://sensu-backend-02:2380 \
+--data-dir /var/lib/sensu/sensu-backend/etcd/data \
+--wal-dir /var/lib/sensu/sensu-backend/etcd/wal 
+
+ETCDCTL_API=3 etcdctl snapshot restore snapshot.db \
+--name sensu-backend-03 \
+--initial-cluster sensu-backend-01=http://sensu-backend-01:2380,sensu-backend-02=http://sensu-backend-02:2380,sensu-backend-03=http://sensu-backend-03:2380 \
+--initial-cluster-token sensu-backend-03 \
+--initial-advertise-peer-urls http://sensu-backend-03:2380 \
+--data-dir /var/lib/sensu/sensu-backend/etcd/data \
+--wal-dir /var/lib/sensu/sensu-backend/etcd/wal 
```

Comment:
The `data-dir` and `wal-dir` values _should_ probably change here. There's some other bits that will need to change. For example, removing the etcd dirs earlier. Doing that doesn't allow the backends to start back up.